### PR TITLE
Add icon, tooltip, rename button in AnimationLibraryEditor for clarity.

### LINKED ISSUE #5565: In `Edit Animation Libraires` window rename `Add Library` to `Create New Library`
### Describe the project you are working on

CHecking godot 4 animation libraries.

### Describe the problem or limitation you are having in your project

If in AnimationPlayer click on Animation and Manage Animations at the top of the the window for animation libraries there are two buttons: `Add Library` and `Load Library`.
Labels are similar which creates confusion for anyone using the system (not just me, there are already youtube videos with people also mentioning this in their tutorials), if you press the Add Library button you get a pop up window which says `Global library will be created.` since it clearly creates a new library I believe it would be benificial to change the button's label to reflect that a bit better.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Renaming `Add Library` -> `Create New Library` defuses a situation when someone wants to add an existing library and clicks on `Add Library` button which creates a new library but does not add an already exiting one.
The second button `Load Library` serves for the purpose of adding existing libraries instead so the naming is a bit confusing.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Renaming the Label for `Add Library` button.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

It's editor's core.

### Is there a reason why this should be core and not an add-on in the asset library?

It's editor's core.

### PR DESCRIPTION
**Pull request by Illauriel**
- Resolves godotengine/godot-proposals/issues/5565

Renames Add Library to New Library
Adds icons to New Library and Load Library
Adds tooltips to New Library and Load Library

### CODE REVIEW

File: editor/plugins/animation_library_editor.cpp [hunk 1]
```diff
@@ -760,12 +763,14 @@ AnimationLibraryEditor::AnimationLibraryEditor() {
...
-	hb->add_child(b);
-	b = memnew(Button(TTR("Load Library")));
-	b->connect("pressed", callable_mp(this, &AnimationLibraryEditor::_load_library));
-	hb->add_child(b);
+	new_library_button = memnew(Button(TTR("New Library")));
+	new_library_button->set_tooltip_text(TTR("Create new empty library"));
+	new_library_button->connect("pressed", callable_mp(this, &AnimationLibraryEditor::_add_library));
+	hb->add_child(new_library_button);
+	load_library_button = memnew(Button(TTR("Load Library")));
+	load_library_button->set_tooltip_text(TTR("Load library from disk"));
```
Comment (akien-mga) [inline editor/plugins/animation_library_editor.cpp:771]:
```suggestion
	new_library_button->set_tooltip_text(TTR("Create new empty animation library."));
	new_library_button->connect("pressed", callable_mp(this, &AnimationLibraryEditor::_add_library));
	hb->add_child(new_library_button);
	load_library_button = memnew(Button(TTR("Load Library")));
	load_library_button->set_tooltip_text(TTR("Load animation library from disk."));
```

**Review by akien-mga — APPROVED** [review]

File: editor/plugins/animation_library_editor.cpp (hunk 1 above)
Comment (Illauriel) [reply editor/plugins/animation_library_editor.cpp]: Done!

File: editor/plugins/animation_library_editor.cpp [hunk 2]
```diff
@@ -621,6 +621,9 @@ void AnimationLibraryEditor::update_tree() {
 		return;
 	}
 
+	new_library_button->set_icon(get_editor_theme_icon(SNAME("Add")));
+	load_library_button->set_icon(get_editor_theme_icon(SNAME("Load")));
```
Comment (KoBeWi) [inline editor/plugins/animation_library_editor.cpp:625]: This should be in `NOTIFICATION_THEME_CHANGED`.

Comment (Illauriel) [reply editor/plugins/animation_library_editor.cpp:625]: Alright, will look into it later today!^^

Comment (Illauriel) [reply editor/plugins/animation_library_editor.cpp]: Done!

**Review by KoBeWi — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Comment by Illauriel** [comment]
Yaaaay! I'm so happy ❤️

**Pull request merged**